var/spack/repos/builtin/packages/aocl-crypto/package.py

```diff
@@ -35,24 +32,34 @@ class AoclCrypto(CMakePackage):
 
     _name = "aocl-crypto"
     homepage = "https://www.amd.com/en/developer/aocl/cryptography.html"
-    git = "https://github.com/amd/aocl-crypto"
-    url = "https://github.com/amd/aocl-crypto/archive/refs/tags/4.2.tar.gz"
+    url = "https://github.com/amd/aocl-crypto/archive/4.2.tar.gz"
+    git = "https://github.com/amd/aocl-crypto/"
 
     maintainers("amd-toolchain-support")
+
     version(
-        "4.2",
-        sha256="2bdbedd8ab1b28632cadff237f4abd776e809940ad3633ad90fc52ce225911fe",
+        "5.0",
+        sha256="b15e609943f9977e13f2d5839195bb7411c843839a09f0ad47f78f57e8821c23",
         preferred=True,
     )
+    version("4.2", sha256="2bdbedd8ab1b28632cadff237f4abd776e809940ad3633ad90fc52ce225911fe")
 
     depends_on("c", type="build")  # generated
     depends_on("cxx", type="build")  # generated
     variant("examples", default=False, description="Build examples")
+    variant("ipp", default=False, description="Build Intel IPP library")
+
+    # Removed dependency on lsb_release
+    patch(
+        "lsb_release.patch",
+        sha256="b61d6d2518276c56d37e8c64d18488081af70f29a62f315ecbd23664e0e440b9",
```

Comment:
@amd-toolchain-support, this patch does not apply to `aocl-crypto-4.2`:
```py
==> Installing aocl-crypto-4.2-ewtpd5ip36d36iio6jzluxri5rtuml5q [12/12]
==> No binary for aocl-crypto-4.2-ewtpd5ip36d36iio6jzluxri5rtuml5q found: installing from source
==> Fetching https://mirror.spack.io/_source-cache/archive/2b/2bdbedd8ab1b28632cadff237f4abd776e809940ad3633ad90fc52ce225911fe.tar.gz
1 out of 1 hunk FAILED -- saving rejects to file cmake/CompilerLinux.cmake.rej
Reversed (or previously applied) patch detected!  Assume -R? [n]
```
```suggestion
        "lsb_release.patch",
        sha256="b61d6d2518276c56d37e8c64d18488081af70f29a62f315ecbd23664e0e440b9",
        when="@5.0",     
```

Reply:
The issue has been addressed in the most recent commit.

Reply:
Confirmed visually and works for the build!